portfolio/src/main/webapp/map.js

```diff
@@ -14,9 +14,29 @@
 
 class Map {
   constructor() {
+    // Information for each location on the map.
     this.dukeLocation = {lat: 36.001111, lng: -78.938889};
     this.wilsonLocation = {lat: 35.9974, lng: -78.9414};
     this.chapelLocation = {lat: 36.0019, lng: -78.9403};
+    this.keohaneLocation = {lat: 35.9989, lng: -78.9373}
+    this.dukeDescription = '<p class=info-window-description> \
```

Comment:
This is a lot of html to keep in js, break up into html templates.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template

Reply:
+1